Library/common.props

```diff
@@ -6,9 +6,19 @@
     
     <PackageProjectUrl>https://github.com/DiscUtils/DiscUtils</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/DiscUtils/DiscUtils/blob/master/LICENSE.txt</PackageLicenseUrl>
-    <RepositoryUrl>https://github.com/DiscUtils/DiscUtils</RepositoryUrl>
```

Comment:
RepositoryUrl is not needed?

Reply:
Correct, SourceLink will automatically populate `RepositoryUrl` for you. One of the nice side-effects is that if someone forks this project, `RepositoryUrl` gets updated automatically and points to their fork.